grr/core/grr_response_core/lib/casing.py

```diff
@@ -0,0 +1,25 @@
+# Lint as: python3
+"""Functions to convert strings between different case styles."""
+import re
+
+
+def SnakeToCamel(snake_str: str) -> str:
+  """Convert a string in snake_case to lowerCamelCase."""
```

Comment:
I would make a note that these functions assume that the given string is a single identifier and the behaviour for string with spaces is undefined.